src/interfaces/token.ts

```diff
@@ -118,7 +126,21 @@ export class TokenInterface<C extends ChainId> extends ServiceInterface<C> {
       }
     }
 
-    if ([1, 1337, 250, 42161].includes(this.chainId)) {
+    if (isFantom(this.chainId)) {
+      const balance = await this.ctx.provider.read.getBalance(account);
+      const priceUsdc = await this.yearn.services.oracle.getPriceUsdc(WrappedFantomAddress);
+      balances.sdk = [
+        {
+          address: account,
+          token: FANTOM_TOKEN,
+          balance: balance.toString(),
+          balanceUsdc: new BigNumber(balance.toString()).times(new BigNumber(priceUsdc)).toString(),
```

Comment:
```suggestion
          balanceUsdc: new BigNumber(balance.toString()).div(10**18).times(new BigNumber(priceUsdc)).toString(),
```
Should it be like this so that balanceUsdc is expressed in Usdc?

Reply:
Correct, fixed with https://github.com/yearn/yearn-sdk/pull/262/commits/7d0806ba67870cc56102db636208b9622214c883